Test/git-issues/git-issue-622$f.dfy

```diff
@@ -0,0 +1,11 @@
+// RUN: rm -f 'git-issue-622$f/git_issue_622$f.java'
```

Comment:
This line is not necessary, since L2 will overwrite the file anyhow. This also means we avoid a dependency on `rm`, which may not be available on Windows systems.